generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -27,7 +34,10 @@ public void execute() {
             state.getRuntime().getAddressSpaceView().gc();
             state.getRuntime().getAddressSpaceView().invalidateClientCache();
             state.getRuntime().getAddressSpaceView().invalidateServerCaches();
-            System.out.println("CheckpointOperation Completed");
+
+//            TODO: uncomment when verification supports it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)